Gemfile

```diff
@@ -4,3 +4,7 @@ source "https://rubygems.org"
 
 # Specify your gem's dependencies in convenient_service.gemspec
 gemspec
+
+group :development, :test do
+  gem "appraisal", github: "thoughtbot/appraisal"
```

Comment:
@nazar-kostyk 

Please, specify a concrete commit as well.

This way CI will be deterministic even when someone merges new code into `thoughtbot/appraisal`.

Thanks.

Reply:
Thanks! Addressed in [9bbe464](https://github.com/marian13/convenient_service/pull/81/commits/9bbe4646001f16d120bf10659f3168ac5c02ce40)